agr_literature_service/api/crud/topic_entity_tag_crud.py

```diff
@@ -222,19 +222,27 @@ def show_source(db: Session, topic_entity_tag_source_id: int):
 def show_all_reference_tags(db: Session, curie_or_reference_id, token: str, page: int = 1,
                             page_size: int = None, count_only: bool = False,
                             sort_by: str = None, desc_sort: bool = False,
-                            species_only: bool = False, species: str = None):
+                            column_filter: str = None, species: str = None):
```

Comment:
The new name is a bit confusing. `column_filter` should be 'column_value_only` or something similar, and `species` should be `column_filter`

Reply:
yes, sounds great! Thanks @sweng66

Reply:
Cool. @valearna

Reply:
Fixed

Reply:
fixed the parameter names